server/command.go

```diff
@@ -113,7 +129,10 @@ func (p *Plugin) executeCommandList(args *model.CommandArgs) {
 	// Deduplicate entries
 	teams := make(map[string]struct{})
 	for _, message := range wecomeMessages {
-		teams[message.TeamName] = struct{}{}
+		teamNameSlice := strings.Split(message.TeamName, ",")
+		for _, tn := range teamNameSlice {
+			teams[tn] = struct{}{}
+		}
```

Comment:
Does spaces need to also be trimmed here?

Reply:
@hanzei  this won't be needed here.